sdk/identity/identity/src/credentials/onBehalfOfCredential.browser.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TokenCredential, AccessToken } from "@azure/core-auth";
+import { credentialLogger, formatError } from "../util/logging";
+
+const credentialName = "OnBehalfOfCredential";
+const BrowserNotSupportedError = new Error(`${credentialName}: Not supported in the browser.`);
+const logger = credentialLogger(credentialName);
+
+export class OnBehalfOfCredential implements TokenCredential {
+  constructor() {
+    logger.info(formatError("", BrowserNotSupportedError));
```

Comment:
What's up with the empty string?

Reply:
This empty string is just the scope. Generally, the use cases of this method pass a scope through, and the scope is far smaller than the message. Moving the scope after the message looks awkward for this specific internal utility, IMO